app/views/inbox/show.html.haml

```diff
@@ -11,4 +11,5 @@
       class: "btn btn-light",
       method: :get,
       params: { last_id: @inbox_last_id, author: @author }.compact,
+      data: { hotkey: "." },
```

Comment:
this only seems to work once

Reply:
Have added a Stimulus controller to deal with this.